cognite_toolkit/_cdf_tk/utils/producer_worker.py

```diff
@@ -97,7 +97,7 @@ def run(self) -> None:
             process_task = progress.add_task(self.process_description.title(), total=self.iteration_count)
             process_thread = threading.Thread(target=self._process_worker, args=(progress, process_task))
 
-            write_task = progress.add_task(self.download_description.title(), total=self.iteration_count)
+            write_task = progress.add_task(self.write_description.title(), total=self.iteration_count)
```

Comment:
This was the bug, used the download description twice instead of the write description.